components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag

```diff
@@ -12,22 +12,30 @@
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
--->
+-->*/%>
 <%
 	var authSuccess = session.get("authSuccess");
-	if(!authSuccess){
+
+		if(!authSuccess){
 		var queryParameters = request.getQueryString();
-		if(queryParameters == "null" || queryParameters == null){
-			response.sendRedirect("login?requestUrl=" + request.getRequestURI());
-		} else{
-			response.sendRedirect("login?requestUrl=" + request.getRequestURI() + "%3F" + request.getQueryString());
-		}
+        	if(queryParameters == "null" || queryParameters == null){
```

Comment:
This code snippet has been moved to reportController.jag and the formatting issue is fixed there.
Fixed with commit 130406a